.buildkite/steps/upload-linux-packages.sh

```diff
@@ -2,7 +2,7 @@
 
 file=${1}
 extension="${file##*.}"
-registry="test-splitter-${extension}"
+registry="test-engine-client-${extension}"
```

Comment:
For thing that aren't already namespaced under "buildkite", I reckon we should call it "buildkite-test-engine-client"

Reply:
Package cloud registry is stored within `buildkite` namespace. See this [example](https://buildkite.com/organizations/buildkite/packages/registries/test-splitter-deb/packages/019106fc-6d20-781b-816e-74c39e3dee99) from the old registry.